org/mozilla/jss/ssl/javax/JSSEngineReferenceImpl.java

```diff
@@ -195,6 +195,12 @@ private void createBufferFD() throws SSLException {
 
         // Initialize ssl_fd from the model Buffer-backed PRFileDesc.
         ssl_fd = SSL.ImportFD(model, fd);
+        if (ssl_fd == null) {
+            throw new SSLException("Error creating SSL socket on top of buffer-backed PRFileDesc.");
+        }
+
+        fd.clear();
+        fd = null;
```

Comment:
`fd` is a local variable, and usually it's not necessary to nullify a local variable, but do you think it's necessary here?

Reply:
It isn't necessary to nullify a local variable to trigger the GC, I agree. 

However, it just ensures that if we were to use `fd` again later without thinking about it, we'd get a NPE rather than a segfault (if we tried to call into the JNI layer using `fd` again and something didn't adequately defend against a cleared pointer).

IMO this is preferable so I nulled it.